lib/sle2docker/native_image.rb

```diff
@@ -34,35 +34,36 @@ def activate
     private
 
     def compute_metadata_file
+      match = parse_image_filename(@image_name)
+      @metadata = parse_metadata_file("#{match['metadata_file']}.metadata")
+
+      @repository   = @metadata['image']['name']
+      @tag          = @metadata['image']['tags'][0]
```

Comment:
What if there's no tag ? I'd prefer then:

```ruby
@tag          = @metadata['image']['tags'][0] || "latest"
```

Reply:
See `parse_metadata_file` method, as it already validates this value.